samples/zoomableapp/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,52 @@
+<LinearLayout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:fresco="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:baselineAligned="false"
+  android:orientation="horizontal"
+  android:paddingBottom="@dimen/activity_vertical_margin"
+  android:paddingLeft="@dimen/activity_horizontal_margin"
+  android:paddingRight="@dimen/activity_horizontal_margin"
+  android:paddingTop="@dimen/activity_vertical_margin"
+  tools:context="com.facebook.zoomableapp.MainActivity"
+  >
+	<android.support.v4.view.ViewPager
```

Comment:
Is there any specific reason behind using a ViewPager here? Would a simple view not suffice?